src/stylesheets/settings/_mixins.scss

```diff
@@ -305,13 +305,7 @@
   }
 }
 @mixin zeta {
-  @include fontsize(18);
-  line-height: 1.167; // 21px;
-
-  @include media($desktop) {
-    @include fontsize(20);
-    line-height: 1.2; // 24px
-  }
+  @include epsilon;
```

Comment:
We should remove the `zeta` mixin and replace all references of it to `epsilon` instead as it has remained as a duplicate.